src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/RestrictedFieldPermissions.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import uk.gov.digital.ho.hocs.casework.security.AccessLevel;
+
+import java.util.Map;
+import java.util.Set;
+
+public class RestrictedFieldPermissions {
+
+    @JsonValue
+    public Map<String, FieldAccessLevels> caseFieldAccessLevels;
+
+    @JsonCreator
```

Comment:
Thankyou for un-lombocking

Reply:
I had it with Lombok but JsonCreator actually needs an AllArgsConstructor to work